internal/logging/keys.go

```diff
@@ -3,6 +3,30 @@
 
 package logging
 
+import (
+	"context"
+
+	baselogging "github.com/hashicorp/aws-sdk-go-base/v2/logging"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
 const (
-	KeyResourceId = "id"
+	HTTPKeyRequestBody  = "http.request.body"
+	HTTPKeyResponseBody = "http.response.body"
+	KeyResourceId       = "id"
 )
+
+// MaskSensitiveValuesByKey masks sensitive values using tflog
+func MaskSensitiveValuesByKey(ctx context.Context, keys ...string) context.Context {
```

Comment:
Still feel uneasy about this but I guess we wait on guidance from upstream?

Reply:
yeah, upstream needs to figure out logging/tracing as well.